test

```diff
@@ -143,6 +144,12 @@ function install_console()
     ws enable console
 )
 
+function remove_console()
+(
+    cd "${path_test}"
+    docker-compose rm --stop console
```

Comment:
this is missing `docker-compose up -d console` from my previous suggestion, so would probably force a rebuild of console still?

Reply:
i.e. you only want to clear the changes to the console container based on what you're doing, so a rm and an up gets it to that state